### PR TITLE
Non ascii support for some strings functions

### PR DESCRIPTION
**Pull request by arkadiont**

### CODE REVIEW

**Comment by juliusrickert** [comment]
@robertkrimen could you take a look at this? This should fix [this issue with DNScontrol](https://github.com/StackExchange/dnscontrol/issues/1092).

**Comment by juliusrickert** [comment]
@deoxxa

File: ast/node.go [hunk 1]
```diff
@@ -10,8 +10,8 @@ node types are concerned) and may change in the future.
 package ast
 
 import (
-	"github.com/robertkrimen/otto/file"
-	"github.com/robertkrimen/otto/token"
+	"github.com/arkadiont/otto/file"
+	"github.com/arkadiont/otto/token"
```
Comment (stevenh) [inline ast/node.go:14]:
```suggestion
	"github.com/robertkrimen/otto/file"
	"github.com/robertkrimen/otto/token"
```

File: ast/walk_example_test.go [hunk 2]
```diff
@@ -4,9 +4,9 @@ import (
 	"fmt"
 	"log"
 
-	"github.com/robertkrimen/otto/ast"
-	"github.com/robertkrimen/otto/file"
-	"github.com/robertkrimen/otto/parser"
+	"github.com/arkadiont/otto/ast"
+	"github.com/arkadiont/otto/file"
+	"github.com/arkadiont/otto/parser"
```
Comment (stevenh) [inline ast/walk_example_test.go:9]:
```suggestion
	"github.com/robertkrimen/otto/ast"
	"github.com/robertkrimen/otto/file"
	"github.com/robertkrimen/otto/parser"
```

File: ast/walk_test.go [hunk 3]
```diff
@@ -4,9 +4,9 @@ import (
 	"log"
 	"testing"
 
-	"github.com/robertkrimen/otto/ast"
-	"github.com/robertkrimen/otto/file"
-	"github.com/robertkrimen/otto/parser"
+	"github.com/arkadiont/otto/ast"
+	"github.com/arkadiont/otto/file"
+	"github.com/arkadiont/otto/parser"
```
Comment (stevenh) [inline ast/walk_test.go:9]:
```suggestion
	"github.com/robertkrimen/otto/ast"
	"github.com/robertkrimen/otto/file"
	"github.com/robertkrimen/otto/parser"
```

File: builtin_function.go [hunk 4]
```diff
@@ -2,11 +2,10 @@ package otto
 
 import (
 	"fmt"
+	"github.com/arkadiont/otto/parser"
```
Comment (stevenh) [inline builtin_function.go:5]:
```suggestion
	"github.com/robertkrimen/otto/parser"
```

File: cmpl.go [hunk 5]
```diff
@@ -1,8 +1,8 @@
 package otto
 
 import (
-	"github.com/robertkrimen/otto/ast"
-	"github.com/robertkrimen/otto/file"
+	"github.com/arkadiont/otto/ast"
+	"github.com/arkadiont/otto/file"
```
Comment (stevenh) [inline cmpl.go:5]:
```suggestion
	"github.com/robertkrimen/otto/ast"
	"github.com/robertkrimen/otto/file"
```

File: cmpl_evaluate_expression.go [hunk 6]
```diff
@@ -2,10 +2,9 @@ package otto
 
 import (
 	"fmt"
+	"github.com/arkadiont/otto/token"
```
Comment (stevenh) [inline cmpl_evaluate_expression.go:5]:
```suggestion
	"fmt"
	"github.com/robertkrimen/otto/token"
```

File: cmpl_evaluate_statement.go [hunk 7]
```diff
@@ -2,9 +2,8 @@ package otto
 
 import (
 	"fmt"
+	"github.com/arkadiont/otto/token"
```
Comment (stevenh) [inline cmpl_evaluate_statement.go:5]:
```suggestion
	"fmt"
	"github.com/robertkrimen/otto/token"
```

File: cmpl_parse.go [hunk 8]
```diff
@@ -4,9 +4,9 @@ import (
 	"fmt"
 	"regexp"
 
-	"github.com/robertkrimen/otto/ast"
-	"github.com/robertkrimen/otto/file"
-	"github.com/robertkrimen/otto/token"
+	"github.com/arkadiont/otto/ast"
+	"github.com/arkadiont/otto/file"
+	"github.com/arkadiont/otto/token"
```
Comment (stevenh) [inline cmpl_parse.go:9]:
```suggestion
	"github.com/robertkrimen/otto/ast"
	"github.com/robertkrimen/otto/file"
	"github.com/robertkrimen/otto/token"
```

File: cmpl_test.go [hunk 9]
```diff
@@ -3,7 +3,7 @@ package otto
 import (
 	"testing"
 
-	"github.com/robertkrimen/otto/parser"
+	"github.com/arkadiont/otto/parser"
```
Comment (stevenh) [inline cmpl_test.go:6]:
```suggestion
	"github.com/robertkrimen/otto/parser"
```

File: dbg.go [hunk 10]
```diff
@@ -3,7 +3,7 @@
 package otto
 
 import (
-	Dbg "github.com/robertkrimen/otto/dbg"
+	Dbg "github.com/arkadiont/otto/dbg"
```
Comment (stevenh) [inline dbg.go:6]:
```suggestion
	Dbg "github.com/robertkrimen/otto/dbg"
```

File: error.go [hunk 11]
```diff
@@ -4,7 +4,7 @@ import (
 	"errors"
 	"fmt"
 
-	"github.com/robertkrimen/otto/file"
+	"github.com/arkadiont/otto/file"
```
Comment (stevenh) [inline error.go:7]:
```suggestion
	"github.com/robertkrimen/otto/file"
```

File: error_native_test.go [hunk 12]
```diff
@@ -33,7 +33,7 @@ func TestErrorContextNative(t *testing.T) {
 		is(err1.message, "wow")
 		is(len(err1.trace), 3)
 		is(err1.trace[0].location(), "F (test.js:2:29)")
-		is(err1.trace[1].location(), "github.com/robertkrimen/otto.TestErrorContextNative.func1.1 (error_native_test.go:15)")
+		is(err1.trace[1].location(), "github.com/arkadiont/otto.TestErrorContextNative.func1.1 (error_native_test.go:15)")
```
Comment (stevenh) [inline error_native_test.go:36]:
```suggestion
		is(err1.trace[1].location(), "github.com/robertkrimen/otto.TestErrorContextNative.func1.1 (error_native_test.go:15)")
```

File: evaluate.go [hunk 13]
```diff
@@ -5,7 +5,7 @@ import (
 	"math"
 	"strings"
 
-	"github.com/robertkrimen/otto/token"
+	"github.com/arkadiont/otto/token"
```
Comment (stevenh) [inline evaluate.go:8]:
```suggestion
	"github.com/robertkrimen/otto/token"
```

File: function_stack_test.go [hunk 14]
```diff
@@ -16,8 +16,8 @@ func TestFunction_stack(t *testing.T) {
 		vm.Run(s)
 
 		expected := []_frame{
-			{native: true, nativeFile: "function_stack_test.go", nativeLine: 30, offset: 0, callee: "github.com/robertkrimen/otto.TestFunction_stack.func1.2"},
-			{native: true, nativeFile: "function_stack_test.go", nativeLine: 25, offset: 0, callee: "github.com/robertkrimen/otto.TestFunction_stack.func1.1"},
+			{native: true, nativeFile: "function_stack_test.go", nativeLine: 30, offset: 0, callee: "github.com/arkadiont/otto.TestFunction_stack.func1.2"},
+			{native: true, nativeFile: "function_stack_test.go", nativeLine: 25, offset: 0, callee: "github.com/arkadiont/otto.TestFunction_stack.func1.1"},
```
Comment (stevenh) [inline function_stack_test.go:20]:
```suggestion
			{native: true, nativeFile: "function_stack_test.go", nativeLine: 30, offset: 0, callee: "github.com/robertkrimen/otto.TestFunction_stack.func1.2"},
			{native: true, nativeFile: "function_stack_test.go", nativeLine: 25, offset: 0, callee: "github.com/robertkrimen/otto.TestFunction_stack.func1.1"},
```

**Review by stevenh — CHANGES_REQUESTED** [review]
Could you rebase to master and replace all the references to arkadiont with robertkrimen

I've detailed some example below.

**Comment by stevenh** [comment]
Closing in favour of #423

**Pull request closed without merging**